.mergify.yml

```diff
@@ -101,6 +101,17 @@ pull_request_rules:
         - files~=^\.mergify\.yml$
     actions:
       delete_head_branch:
+  - name: notify the backport has not been merged yet
+    conditions:
```

Comment:
Nit. Can we also check if the backport is older than some days? To avoid receiving notifications of backports you have just created.

Reply:
Yes, there is also a condition for that. I'll test it beforehand. Do you want me to add it here or in a follow up?

Reply:
As you prefer, not so important, I was thinking in something like what you are doing here: https://github.com/elastic/apm-pipeline-library/pull/1381/files

In any case I guess that this will be a problem only for people opening backports during the weekend :D